### ISSUE TITLE
Ist the game Mobile friendly

### ISSUE THREAD

**Issue by wumix** [Fernando-Marquardt/artemis#1]
Hello my question is your posted game Mobile friendly?

**Comment by fernando-marquardt** [comment]
No, currently it is not mobile friendly. It may become in the future, but the priority right now is to update the code and make it run.
